get-config.js

```diff
@@ -0,0 +1,21 @@
+function readConfig() {
+	try {
+		return require( './config' )();
```

Comment:
@beaucollins I'm not sure what kind of config file you are using, but the one I had (which matches what's in the project's README needs to be executed in order to return its data. Therefore I went ahead and turned this into a call on the import. If the file is missing or if we could not read it then this will fail and kill the build process.